File: aat-android/src/main/java/ch/bailu/aat/preferences/location/SolidMockLocationFile.java
package ch.bailu.aat.preferences.location;

import android.content.Context;

import ch.bailu.aat.preferences.Storage;
import ch.bailu.aat_lib.preferences.SolidString;

public class SolidMockLocationFile extends SolidString {
    private final static String KEY="mock_file";


    public SolidMockLocationFile(Context c) {
        super(new Storage(c), KEY);
    }

}

File: aat-lib/src/main/java/ch/bailu/aat_lib/preferences/location/SolidMockLocationFile.kt
package ch.bailu.aat_lib.preferences.location

import ch.bailu.aat_lib.preferences.SolidString
import ch.bailu.aat_lib.preferences.StorageInterface

class SolidMockLocationFile(storage: StorageInterface) : SolidString(storage, KEY) {
    companion object {
        private const val KEY = "mock_file"
    }
}
